words/words.md

```diff
@@ -475,29 +475,27 @@ Der Pessimist erfindet den Fallschirm.
 
 Was wäre, wenn R<sub>0</sub> viel größer wäre als wir glauben und wir es trotz der Maßnahmen nicht schaffen würden, R auf unter 1 zu drücken? 
 
-Selbst wenn wir es nicht schaffen, R < 1 zu erreichen, reduziert das Drücken dieser Reproduktionszahl die Anzahl der Fälle und rettet damit Leben. Nichtsdestotrotz bleibt R < 1 das Ideal und hier sind ein paar weitere Möglichkeiten, dieses zu erreichen: 
+Selbst wenn wir es nicht schaffen, R < 1 zu erreichen, reduziert das Drücken der Reproduktionszahl die Anzahl der Fälle und rettet damit Leben. Nichtsdestotrotz bleibt R < 1 das Ideal und hier sind ein paar weitere Möglichkeiten, wie wir das schaffen können:  
```

Comment:
Ich finde deine Änderungen in 478 gut, schlage aber vor, falls das nicht schon diskutiert worden ist, das "Drücken" durch "Senken" zu ersetzen. "Drücken" gibt einen Energieaufwand vor, aber nicht die Richtung.

Reply:
+1